src/client/drafting/createdraft.ts

```diff
@@ -142,38 +142,128 @@ export const getDraftFormat = (params: DraftParams, cube: Cube): DraftFormat =>
   return createDefaultDraftFormat(params.packs, params.cards || 15);
 };
 
-const createPacks = (format: DraftFormat, seats: number, nextCardFn: NextCardFn): CreatePacksResult => {
-  let ok = true;
-  let messages: string[] = [];
```

Comment:
The jest tests were failing because messages was always empty in the result even when the stub nextCardFn in my test returned messages. On further reflection I think both messages and ok would never be correctly set on the result object, but that wasn't noticeable because the actual nextCardFn implementation throws an Error when it cannot find a card, thus bypassing result being returned in an error state.

Also exacerbated by the fact that the method returned a variable named "result" and the return of nextCardFn was assigned to "results" as well. This I improved by renaming the latter to cardResult.